examples/esm/index.ts

```diff
@@ -0,0 +1,22 @@
+import "reflect-metadata";
+import path from "node:path";
+import url from "node:url";
+import { ApolloServer } from "@apollo/server";
+import { startStandaloneServer } from "@apollo/server/standalone";
+import { buildSchema } from "type-graphql";
+import { RecipeResolver } from "./recipe.resolver.js";
+
+// Build TypeGraphQL executable schema
+const schema = await buildSchema({
+  // Array of resolvers
+  resolvers: [RecipeResolver],
+  // Create 'schema.graphql' file with schema definition in current directory
+  emitSchemaFile: path.resolve(path.dirname(url.fileURLToPath(import.meta.url)), "schema.graphql"),
```

Comment:
Maybe it's easier to use `process.cwd()`?

Reply:
No. `process.cwd()` returns the current working directory, not the folder of the current file and thus the path to the schema file.

That said, loading schema files is weird to me - to me the purpose of type-graphql is that you can define the GraphQL schema via decorators on class members without having to resort to schema files that are not near the code and thus are harder to maintain.

Reply:
@kf6kjg Unfortunately, neither vscode graphql nor graphql code generator can't read `GraphQLSchema` and spawning a whole server for introspection is a much bigger overkill than `emitSchemaFile`.

> schema files that are not near the code and thus are harder to maintain
I'm not sure if you know that the SDL file is a generated artifact that represents the built schema. It's not read or used at all at runtime, it's just for documentation purposes and for other ecosystem tools to cooperate.

Reply:
@kf6kjg 
> No. process.cwd() returns the current working directory, not the folder of the current file and thus the path to the schema file.

You're right! But you just haven't thought about the use case at all.
We can just point this in examples:
```ts
emitSchemaFile: path.resolve(process.cwd(), "./src/api/schema.graphql"),
```
So the schema would be emitted in the right folder, assuming the app is run from the root folder.